### PR TITLE
[TS2E] Improve logging

### PR DESCRIPTION
**Pull request by justinchuby**
Serializing the text without having to do so can be costly for large outputs like ExportedProgram

### CODE REVIEW

File: torch/_export/converter.py [hunk 1]
```diff
@@ -1001,12 +1001,12 @@ def convert(self) -> ExportedProgram:
             self.name_to_non_tensor_attributes,
         )
         gm = graph_converter.convert()
-        log.info(f"GraphModule: {gm.print_readable(print_output=False)}")  # noqa: G004
+        log.info("GraphModule: %s", gm.print_readable(print_output=False))
```
Comment (justinchuby) [inline torch/_export/converter.py:1004]: Ideally we call this lazily too

**Review by ydwu4 — APPROVED** [review]

**Pull request closed without merging**